docs/en/status-communities/invite-people-to-a-channel.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 286
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Invite people to a channel
+
+Whether you're a channel owner or a member, you can tell people about the channel and invite them to join.
+
+!!! note
+    You can't join a channel if you are not a member of a community.
+
+## Invite options
+
+There are three options for inviting someone to a channel.
+
+| Option | Description | Works with |
+|---------|-------------|------|
+| Share to Status contacts | Use this option to share the channel with people in your Status contact list. | Mobile and desktop |
+| Show QR code | Use this option to show the channel QR code to people to scan. | Mobile |
+| Share a link | Use this option to share a link via your other social media or copy it and share it manually. | Mobile and desktop |
+
+## Invite people to a channel
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-communities: **Communities**.
+    1. Tap the community with the channel you want to share.
+    1. In the channel list, long press the channel you want to share.
+    1. In the action sheet, choose an option and proceed as follows:
+
+    | Option | Steps |
+    |---------|--------|
+    | Share to Status contacts | 1. Select :mobile-add-user: **Invite people from contacts list**. <br/> 2. Check the boxes next to the contacts you want to share with. <br/> 3. Tap **Invite users**. |
+    | Show QR code | 1. Select :moblie-qr-code: **Show QR code**. <br/> 2. Show the QR code to people you want to invite. |
+    | Share a link | 1. Select :mobile-share: **Share channel**. <br/> 2. Share the channel link using your preferred method. 
```

Comment:
'Share the channel link via social media, messenger apps or any other app you prefer.'